commit 8446c9dc92144a7d71af44a549a3c0f39f3160f1
Merge: e4d2138 3eacb4c
Author: Bernhard Kubicek <kubicek@gmx.at>
Date:   Wed Apr 25 22:23:31 2012 -0700

    Merge pull request #152 from loganb/Marlin_v1
    
    Fix compile error on boards without a bed heater.
    
    I could not find why this problem was introduced.
    Its quite obvious, so I think its in general interest to merge.

diff --git a/Marlin/temperature.cpp b/Marlin/temperature.cpp
index 2180f80..783aa23 100644
--- a/Marlin/temperature.cpp
+++ b/Marlin/temperature.cpp
@@ -725,7 +725,9 @@ void min_temp_error(uint8_t e) {
 }
 
 void bed_max_temp_error(void) {
+#if HEATER_BED_PIN > -1
   WRITE(HEATER_BED_PIN, 0);
+#endif
   if(IsStopped() == false) {
     SERIAL_ERROR_START;
     SERIAL_ERRORLNPGM("Temperature heated bed switched off. MAXTEMP triggered !!");
